tests/unittest/_torch/visual_gen/test_attention_perf.py

```
@@ -834,6 +853,191 @@ def test_fa4_cross_attn_quick(
+        """Verify every sage config produces a valid timing result at quick sizes."""
+        if sage_cfg.qk_int8 and torch.cuda.get_device_capability()[1] == 3:
+            pytest.skip("SM103 does not have Int8 Tensor Cores.")
+
+        result = self._bench(batch, num_heads, seq_len, head_dim, sage_cfg)
+        assert result is not None, f"sage {cfg_id} failed for {desc}"
+        assert result["avg_ms"] > 0
+        print(
+            f"  sage {cfg_id} ({desc}): avg={result['avg_ms']:.3f}ms  p95={result['p95_ms']:.3f}ms"
+        )
+
+    @pytest.mark.parametrize("sage_cfg,cfg_id", zip(_SAGE_CONFIGS, _SAGE_CONFIG_IDS))
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)